docs/get-details/dapps/smart-contracts/apps/index.md

```diff
@@ -10,7 +10,7 @@ See the [*TEAL Reference Guide*](../../avm/teal/specification.md) to understand
     Where possible, TEAL code snippets are accompanied by their counterparts in PyTeal. Here are a few things to be aware of when comparing across these two languages:
      
     - Each PyTeal code snippet ends with the action to compile the program and write it to a file so the user can view the underlying TEAL code.
-    - Sometimes the compiled version of a PyTeal code snippet will differ slightly from the TEAL version. However, the resulting function should be equivalent for the particular area of focus in the documentation. In larger more complex programs, this may not always be the case.
+    - Sometimes the compiled version of a PyTeal code snippet will differ slightly from the program version. However, the resulting function should be equivalent for the particular area of focus in the documentation. In larger more complex programs, this may not always be the case.
```

Comment:
@barnjamin Optionally and out-of-band to this PR, leaving a note to highlight a potential, future doc enrichment:  The highlighted section _may_ benefit from mentioning the compiler version pragma ( https://github.com/algorand/pyteal/pull/429).

Reply:
This one I'd purposely left since the version referenced here is not related to the teal/avm version as I read it but the resulting output from teal vs the pyteal if that makes sense

Reply:
@barnjamin Sorry - upon 2nd read, I agree with your assessment.  I'll revert.

Reply:
```suggestion
    - Sometimes the compiled version of a PyTeal code snippet will differ slightly from the TEAL version. However, the resulting function should be equivalent for the particular area of focus in the documentation. In larger more complex programs, this may not always be the case.
```